app/test_main.py

```diff
@@ -1,3 +1,22 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, result_convert_to_human_age",
+        [
+            (14, 14, [0, 0]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (28, 29, [3, 3]),
+            (100, 100, [21, 17])
+        ]
+    )
+    def test_calculate_human_age(
+            self,
+            cat_age: int,
+            dog_age: int,
+            result_convert_to_human_age: list
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == result_convert_to_human_age
```

Comment:
check the ckecklist 
![image](https://user-images.githubusercontent.com/103201010/207107143-2853d884-af72-4aca-9f96-81da533defeb.png)